web/source/keymanweb.js

```diff
@@ -2829,20 +2930,14 @@ if(!window['tavultesoft']['keymanweb']['initialized']) {
         if(Levent.initKeyEvent)
         {
           Levent.initKeyEvent('keypress',true,true,null,false,false,false,false,0,32);
-          var LkeyPress = keymanweb._KeyPress; 
+          Levent._kmw_block = true; // Tell keymanweb._KeyPress to ignore this simulated event.
           
-          /**
-           * Function     _KeyPress
-           * Scope        Private
-           * @param       {Event}     e     event         
-           * Description  Temporarily disable keypress event handling  TODO: does this really work??? objects are passed by reference so should be OK
-           */       
-          keymanweb._KeyPress = function(e) {};
+          // Nasty problems arise from the simulated keystroke if we don't temporarily block the handler.
           Pelem.dispatchEvent(Levent);
           Levent=document.createEvent('KeyboardEvent');
           Levent.initKeyEvent('keypress',true,true,null,false,false,false,false,8,0);
+          Levent._kmw_block = true; // Tell keymanweb._KeyPress to ignore this simulated event.
```

Comment:
Isn't this a duplicate of line 2933?

Reply:
Yup.  Different initialized event object - or at least, event values - so I'm making sure it didn't magically get initialized away in that process.

Turns out it's using a deprecated method, so I'll go make an issue out of that.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/initKeyEvent